text/0119-dynamic-component-creation.md

```diff
@@ -309,27 +308,31 @@ Full details can be found [here](https://github.com/salesforce/lwc/pull/3204)
 
 ## Adoption strategy
 
-This is a breaking change for existing user of `lwc:dynamic` as it supersedes the directive.  However, `lwc:dynamic` directive has not reached GA and all consumers of the directive are internal.  They will need to switch their components to change the tag names to `lwc.dynamic` and the directives from `lwc:dynamic` to `lwc:is`.
+This is a breaking change for existing user of `lwc:dynamic` as it supersedes the directive.  However, `lwc:dynamic` directive has not reached GA and all consumers of the directive are internal.  They will need to switch their components to change the tag names to `lwc:dynamic` and the directives from `lwc:dynamic` to `lwc:is`.
 
 The behavior of `lwc:is` is the same as `lwc:dynamic` when the constructor is not provided, both will not render the dynamic component of its children. The behavior is different when the components are rendered though, as `lwc:is` will render the custom element name rather than the custom element name used with `lwc:dynamic`.  Component authors may need to adjust any testing that relies on this.
 
+To help with the transition, the LWC team will support both `lwc:dynamic` and `lwc:is` directives for existing consumers.  Going forward, the compiler will report a warning when usage of the `lwc:dynamic` directive is detected until it is fully deprecated.
+
 # How we teach this
 
-This concept is fairly intuitive and is similar to the same concepts in other frameworks (Svelte, Vue, Angular).  Providing documentation around how the to use `<lwc.dynamic>` and `lwc:is` with a small playground example should be sufficient.
+This concept is fairly intuitive and is similar to the same concepts in other frameworks (Svelte, Vue, Angular).  Providing documentation around how the to use `<lwc:dynamic>` and `lwc:is` with a small playground example should be sufficient.
 
 Examples of [documentation](https://svelte.dev/docs#template-syntax-svelte-component) and [playground](https://svelte.dev/tutorial/svelte-component) from Svelte.
 
 # Unresolved questions
 
 Will there be issues with collisions?  For example if a dynamic constructor is imported but has the same custom element name as a custom element that's already on the page?
-    - Does the `upgradeableCallback` take care of this?
\ No newline at end of file
+    - Does the `upgradeableCallback` take care of this?
+
+Should we log errors when component authors try to use `<lwc:*>` incorrectly?  For example should we prevent component authors from doing something like `document.createElement('lwc:other');`?
```

Comment:
I would recommend staying away from global patching. This opens the door for all sorts of bugs and it is hard to maintain in the long run (our old friend synthetic shadow DOM). 
I would recommend explicitly mentioning in the documentation that `<lwc:dynamic>` is an LWC compiler signal, and doesn't have any bearing at runtime.

Reply:
Added a [note](https://github.com/salesforce/lwc-rfcs/pull/71/files#diff-a1de05148a61ea9b00f70bd4ff5feb3895c16dd99707b173ba57c081b859250bR412) to address this.